.eslintrc.js

```diff
@@ -2,6 +2,7 @@ const clientESLintConfig = require('./config/eslint');
 
 module.exports = Object.assign({}, clientESLintConfig, {
   env: Object.assign({}, clientESLintConfig.env, {
-    node: true
+    node: true,
+    jest: true,
```

Comment:
Well, then all `desctibe` usages in code will be marked as warning. 
The easiest solution is adding `/*eslint-env jest:true*/` at the top of each test file